### DISCUSSION TITLE
Running yarn dev-server results in Error TS2345: Argument of type 'Signer | Provider' is not assignable

### DISCUSSION THREAD

**Discussion by Zivsun12** [hashlips-lab/nft-erc721-collection#418] [Q&A]
Did any one encounter the bellow issue while trying to run
yarn dev-server 

C:\Users\Zivsun\Desktop\NFT\Source\nft-erc721-collection-2.4.4\smart-contract\typechain\factories\Ownable__factory.ts(73,40)
TS2345: Argument of type 'Signer | Provider' is not assignable to parameter of type 'Provider | Signer | undefined'.
Type 'Provider' is not assignable to type 'Provider | Signer | undefined'.

**Comment by liarco** [comment]
Hi @Zivsun12, I already replied to #416. Let's discuss this in one single place to avoid confusion.

Thank you.
